source/calendar/calendar_api.cpp

```diff
@@ -32,6 +32,7 @@ www.navitia.io
 #include "type/pb_converter.h"
 #include "calendar.h"
 #include "ptreferential/ptreferential.h"
+#include "utils/paginate.h"
```

Comment:
Why do you need to include `utils/paginate.h` here ? I can't see any changer in utils.

Reply:
Because it was on ptreferencial.h, but useless in this context